server/routes/wishCards.js

```diff
@@ -27,10 +42,26 @@ const {
 } = require('../utils/defaultItems');
 const { handleError } = require('../helper/error');
 
-const s3 = new AWS.S3({
-  accessKeyId: process.env.AWS_KEY,
-  secretAccessKey: process.env.AWS_SECRET,
-});
+const { getMessageChoices } = require('../utils/defaultMessages');
+
+let s3;
+let s3storage;
+
+if (process.env === 'production') {
```

Comment:
can you change the check to LOCAL_DEVELOPMENT so that s3 is still used on the dev server

Reply:
aside from that process.env will never be production, process.env.NODE_ENV

Reply:
for that we have USE_AWS too, so this one is not really neccessary anyway